features/step_definitions/machine_set.rb

```diff
@@ -4,3 +4,38 @@
   cb_name ||= :machinesets
   cb[cb_name] = BushSlicer::MachineSet.list(user: admin, project: project("openshift-machine-api"))
 end
+
+Given(/^I pick a random machineset to scale$/) do
+  machine_sets = BushSlicer::MachineSet.list(user: admin, project: project("openshift-machine-api"))
+  cache_resources *machine_sets
+
+  @machine_set = machine_sets.shuffle.first
+end
+
+When(/^I scale the machineset by "([^"]*)"$/) do | by |
```

Comment:
I would suggest this step to read:
```
When /^I scale the machineset to ([-+]?)NUMBER$/ do | op, num |
  case op
  when "-"
    replicas = @machine_set.available_replicas - num.to_i
  when "+"
    replicas = @machine_set.available_replicas + num.to_i
  when nil
    replicas = num.to_i
  else
    raise "wrong operation #{op.inspect} supplied"
  end
  ...
end
```

In this way the step can also be used to shorten clean-up too.